packages/container/README.md

```diff
@@ -0,0 +1,24 @@
+# hixme-ui checkbox
```

Comment:
Don't forget to update the README with Container documentation

Reply:
Lots of documentation in your future lol.  Let me know if you want an assist on adding tests or documentation